### PR TITLE
error message prop for dateInput

### PR DESCRIPTION
**Pull request by venky-0710**
https://dateinput-error--5cb6f92366a0080020cba496.chromatic.com/?path=/story/components-input--date

### CODE REVIEW

File: packages/pebble-web/src/components/styles/Date.styles.ts [hunk 1]
```diff
@@ -17,3 +18,15 @@ export const inputStyle = css({
 export const wrapperStyle = css({
   marginBottom: 20
 });
+
+export const errorStyle = css({
+  display: "flex",
+  backgroundColor: colors.red.lightest,
+  color: colors.red.darker,
+  borderRadius: "0px 0px 3px 3px",
```
Comment (zhirzh) [inline packages/pebble-web/src/components/styles/Date.styles.ts:26]: calendar component shouldn't have border radius

File: packages/pebble-web/src/components/styles/Date.styles.ts [hunk 2]
```diff
@@ -17,3 +18,15 @@ export const inputStyle = css({
...
+  fontSize: "12px",
+  fontWeight: 500,
```
Comment (zhirzh) [inline packages/pebble-web/src/components/styles/Date.styles.ts:28]: check `messageStyle` typography in `Input.styles.ts`

File: packages/pebble-web/src/components/styles/Date.styles.ts [hunk 3]
```diff
@@ -17,3 +18,15 @@ export const inputStyle = css({
 export const wrapperStyle = css({
   marginBottom: 20
 });
+
+export const errorStyle = css({
+  display: "flex",
```
Comment (zhirzh) [inline packages/pebble-web/src/components/styles/Date.styles.ts:23]: no need for flex. `errorMessage` div only contains text

File: packages/pebble-web/src/components/styles/Date.styles.ts [hunk 4]
```diff
@@ -17,3 +18,15 @@ export const inputStyle = css({
...
+  backgroundColor: colors.red.lightest,
+  color: colors.red.darker,
+  borderRadius: "0px 0px 3px 3px",
+  fontSize: "12px",
+  fontWeight: 500,
+  alignItems:"center",
+  justifyContent:"center",
+  padding: "11px"
```
Comment (zhirzh) [inline packages/pebble-web/src/components/styles/Date.styles.ts:31]: adjust padding to left-align error message with calendar dates

**Review by zhirzh — CHANGES_REQUESTED** [review]

**Review by SiriusV21 — APPROVED** [review]

**Review by appy9 — APPROVED** [review]

**Review by zhirzh — APPROVED** [review]

**Pull request merged**